tests/translator/input/function_with_deployment_preference_from_parameters.yaml

```diff
@@ -40,6 +40,7 @@ Resources:
       CodeUri: s3://my-bucket/mySanityTestFunction.zip
       DeploymentPreference:
         Enabled: !Ref MyLowerFalseParameter
+        Type: Linear10PercentEvery1Minute
 
```

Comment:
what are these changes?

Reply:
It's a required field, but it's missing. Json schema won't let it pass.